### PR TITLE
Feat/Chip component

### PR DESCRIPTION
**Pull request by parz3val**
## Description

Chip component with the MUI styling, and stories

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: mods/webui/src/stories/chip/Chip.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import  { useEffect, useState } from "react";
```
Comment (psanders) [inline mods/webui/src/stories/chip/Chip.tsx]: Would you remove `useEffect` and `useState` as they are not being used?

File: mods/webui/src/stories/chip/types.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+type ChipProps = {
+  onRemove?: (event: React.MouseEvent<HTMLButtonElement>) => void;
+  enabled?: boolean;
+  label: String;
```
Comment (psanders) [inline mods/webui/src/stories/chip/types.ts]: Would you replace the "String" wrapper object with primitive type "string"?

**Review by psanders — CHANGES_REQUESTED** [review]
I added some comments, and otherwise it looks great. I will be happy to merge once the changes are made.

File: mods/webui/src/stories/chip/Chip.tsx (hunk 1 above)
Comment (parz3val) [reply mods/webui/src/stories/chip/Chip.tsx]: done.

File: mods/webui/src/stories/chip/types.ts (hunk 2 above)
Comment (parz3val) [reply mods/webui/src/stories/chip/types.ts]: I've replaced it.

**Review by psanders — APPROVED** [review]
You rock! Thank you for the quick turn around.

**Pull request merged**